### PR TITLE
Validate arguments for createapp command

### LINKED ISSUE #310: Validate arguments for createapp

### PR DESCRIPTION
**Pull request by jamesgeorge007**
## Issue

Closes #310 

> Currently, the user can fire in n number of arguments with `foal createapp` which doesn't make any sense.

### CODE REVIEW

File: packages/cli/src/index.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@
  */
 
 // 3p
+import chalk from 'chalk';
```
Comment (LoicPoullain) [inline packages/cli/src/index.ts]: Same as in https://github.com/FoalTS/foal/pull/307#discussion_r247313481

File: packages/cli/src/index.ts [hunk 2]
```diff
@@ -32,6 +34,10 @@ program
   .command('createapp <name>')
   .description('Creates a new directory with a new FoalTS app.')
   .action((name: string) => {
+    if (args.length > 1) {
+      console.log(chalk.red.bold('\n Kindly provide only one argument as the project name'));
+      process.exit(1);
```
Comment (LoicPoullain) [inline packages/cli/src/index.ts]: Using `process.exit` after a `console.log` may truncate the printed data in some situations (see https://nodejs.org/api/process.html#process_process_exit_code and https://nodejs.org/api/process.html#process_a_note_on_process_i_o). Could we use a `return;` instead?

File: packages/cli/src/index.ts (hunk 1 above)
Comment (jamesgeorge007) [reply packages/cli/src/index.ts]: 👍

File: packages/cli/src/index.ts (hunk 2 above)
Comment (jamesgeorge007) [reply packages/cli/src/index.ts]: 👍

**Comment by jamesgeorge007** [comment]
@LoicPoullain Hope it's fine now 😄

**Pull request merged**